### PR TITLE
lk/deps

### PR DESCRIPTION
**Pull request by lukekarrys**
- chore(deps): @npmcli/template-oss@3.4.1
- deps: @npmcli/map-workspaces@2.0.3
- deps: cacache@16.0.6
- deps: npm-profile@6.0.3
- deps: npmlog@6.0.2
- deps: read-package-json@5.0.1
- deps: glob@8.0.1
- deps: npm-packlist@5.0.2
- deps: semver@7.3.7
- deps: lru-cache@7.8.1
- deps: cli-table3@0.6.2
- fix: normalize win32 paths before globbing

### CODE REVIEW

File: node_modules/cli-table3/package.json
Comment (speedytwenty) [inline node_modules/cli-table3/package.json:3]: I've been looking out for this line because when I published cli-table3 (using [np](https://www.npmjs.com/package/np)), it somehow didn't capture the version bump to `0.6.2` in `package.json` in the `v.0.6.2` tag. It seems this package must autocorrect this and I wanted to drop in to give my 👍 

image

**Comment by lukekarrys** [comment]
This needs to wait until two more patches land and are published:
- https://github.com/npm/cacache/pull/101
- https://github.com/npm/npm-packlist/pull/93

File: .github/workflows/ci-docs.yml
Comment (ruyadorno) [inline .github/workflows/ci-docs.yml]: `user.email` and `user.name` seems to be inverted here

File: node_modules/cli-table3/package.json
Comment (ruyadorno) [reply node_modules/cli-table3/package.json:3]: for reference, in the future if you want to validate something similar in a published package, you can use `npm diff`:

```sh
$ npm diff --diff=cli-table3@0.6.1 --diff=cli-table3@0.6.2 package.json
```

will give you the diff (for the `package.json` file) between these two published versions of the package from the registry

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**
